### PR TITLE
Rewrite PollEditor to React-final-form

### PR DESCRIPTION
**Pull request by ollfkaih**
# Description

Literally the title

# Result

No visual changes 🤞

# Testing

- [ ] I have thoroughly tested my changes.

---

Resolves ABA-56

### CODE REVIEW

File: app/routes/polls/components/PollEditor.tsx [hunk 1]
```diff
@@ -23,32 +25,29 @@ import type {
...
   //TODO add reduxForm typing
   editing: boolean;
   initialValues: PollEntity;
   pollId: ID;
   deletePoll: () => Promise<any>;
   toggleEdit: () => void;
 };
-type FieldArrayPropTypes = {
+/*type FieldArrayPropTypes = {
   fields: fieldArrayFieldsPropTypes;
```
Comment (ivarnakken) [inline app/routes/polls/components/PollEditor.tsx:44]: 👀

File: app/routes/polls/components/PollEditor.tsx [hunk 2]
```diff
@@ -187,27 +108,108 @@ const onSubmit = (
...
+        {({ handleSubmit, pristine, submitting }) => (
+          <Form onSubmit={handleSubmit}>
+            <Field
+              name="title"
+              label="Spørsmål"
+              placeholder="Hva er din favorittrett?"
+              component={TextInput.Field}
+              required
+            />
+            <span />
```
Comment (ivarnakken) [inline app/routes/polls/components/PollEditor.tsx:150]: :eyes:

**Review by ivarnakken — APPROVED** [review]

**Review by LudvigHz — APPROVED** [review]
LGTM

File: app/routes/polls/components/PollEditor.tsx [hunk 3]
```diff
@@ -22,17 +23,15 @@ import type { ReactNode } from 'react';
 import type {
   fieldArrayMetaPropTypes,
   fieldArrayFieldsPropTypes,
-} from 'redux-form';
+} from 'react-final-form';
```
Comment (ivarnakken) [inline app/routes/polls/components/PollEditor.tsx:26]: `react-final-form` does not export these types. However, I don't seem to find them in `redux-form` neither ... Has this ever worked?

Comment (ollfkaih) [reply app/routes/polls/components/PollEditor.tsx:26]: Probably not 😕

Comment (eikhr) [reply app/routes/polls/components/PollEditor.tsx:26]: Probably existed in the flow type definitions

**Pull request merged**